Mathlib/CategoryTheory/Galois/Examples.lean

```diff
@@ -102,7 +102,7 @@ instance : GaloisCategory (Action FintypeCat (MonCat.of G)) where
 
 /-- The `G`-action on a connected finite `G`-set is transitive. -/
 theorem Action.pretransitive_of_isConnected (X : Action FintypeCat (MonCat.of G))
-    [IsConnected X] : MulAction.IsPretransitive G X.V where
+    [PreGaloisCategory.IsConnected X] : MulAction.IsPretransitive G X.V where
```

Comment:
Is this change necessary?

Reply:
Yes, because otherwise there's ambiguity between `CategoryTheory.IsConnected` and `CategoryTheory.PreGaloisCategory.IsConnected` :confused:

Reply:
I seem to be able to undo these changes without problems on your current commit, probably because whatever import changed to pull in `CategoryTheory.IsConnected` into this file is now no longer there after the code review.